credentials/src/main/kotlin/SSI.kt

```diff
@@ -270,30 +273,26 @@ public object VerifiablePresentation {
    */
   @Throws(Exception::class)
   public fun create(signOptions: SignOptions, createVpOptions: CreateVpOptions): VpJwt {
+    val presentationSubmission = generatePresentationSubmissionFrom(
+      createVpOptions.presentationDefinition,
+      createVpOptions.verifiableCredentialJwts
+    )
 
-    val usableVcJwts: List<VcJwt> =
-      selectFrom(createVpOptions.presentationDefinition, createVpOptions.verifiableCredentialJwts)
-
-    if (usableVcJwts.size == 0) {
-      throw Exception("There are no useable Vcs that correspond to the presentation definition")
-    }
-
-    // TODO change to be more than one VC
-    val vcToUse: VerifiableCredentialType =
-      FromJwtConverter
-        .fromJwtVerifiableCredential(JwtVerifiableCredential.fromCompactSerialization(usableVcJwts.get(0)))
+    val properties: MutableMap<String, Any> = HashMap()
+    properties["verifiableCredential"] = createVpOptions.verifiableCredentialJwts
+    properties["presentation_submission"] = presentationSubmission
```

Comment:
Use `mapOf` instead, so the map is immutable. 

Better yet, create a type for the container, instead of using a `Map<String, Any>`.

Reply:
yup I'll use mapOf

I think the object still needs to be Map<String, Any> because thats what .properties() accepts on the builder